### DISCUSSION TITLE
Bug? Images losing prompt input

### DISCUSSION THREAD

**Discussion by mattstobber** [thundaga/SD-webui-txt2img-script#26] [General]
I have a batch of images in a folder and randomly some of the images will be a blank gray output because it isn't taking the positive/negative prompts from the input. If I drag the original image to PNG Info, it has the prompt so I know the data exists there.

**Comment by mattstobber** [comment]
I forgot to mention, if I remove the Prompt and Negative Prompt from the "Assign from input image" section then it works fine(since its using the seed from the image). This means that in order to use this script properly I basically have to rely on not changing the prompt in the ui. This is still a MASSIVE time saver and I love this script, but this bug makes it slightly less usable. Thank you for any help.

**Reply by OPCOXE** [reply]
First of all there is the issue section for that ...
Then can you provide the meta data of the image that doesn't work ?

**Reply by mattstobber** [reply]
Sorry. Created issue and posted image.
